packages/web-components/fast-foundation/src/tree-item/tree-item.ts

```diff
@@ -117,7 +104,7 @@ export class FASTTreeItem extends FASTElement {
     protected itemsChanged(oldValue: unknown, newValue: HTMLElement[]): void {
         if (this.$fastController.isConnected) {
             this.items.forEach((node: HTMLElement) => {
-                if (isTreeItemElement(node)) {
+                if (FASTTreeItem.isFASTTreeItemElement(node)) {
```

Comment:
If you wanted to shorten this you could do `FASTTreeItem.is(node)`. Whatever naming convention we use here, I'd like us to use consistently where we have other situations like this.

Reply:
+1 for `is()`